config/initializers/apipie.rb

```diff
@@ -27,6 +27,8 @@
     trans
   end
   config.help_layout = 'apipie_dsl/apipie_dsls/help.html.erb'
+  config.default_class_description = ->(model) { _("A class representing %s object") % model.human }
```

Comment:
```suggestion
  config.default_class_description = ->(model) do
    return nil unless model.respond_to?(:model_name)
    _("A class representing %s object") % model.model_name.human
  end
```

It appears that `model` doesn't necessary respond to `human`, but rather should respond (in case it's `ActiveRecord`) to `model_name`, which returns some wrapper, which responds to `human`.